Resources/Prototypes/Reagents/Food/condiments.yml

```
@@ -0,0 +1,79 @@
+- type: reagent
+  id: Astrotame
+  name: Astrotame
+  desc: The sweetness of a thousand sugars but none of the calories.
+#  physicalDesc:
+  color: teal
+  metabolism:
+    - !type:DefaultFood
+      rate: 1
+
+- type: reagent
+  id: BbqSauce
+  name: BBQ sauce
+  desc: Hand wipes not included.
+  physicalDesc: Gloopy.
+  color: darkred
+  metabolism:
+    - !type:DefaultFood
+      rate: 1
+
+- type: reagent
+  id: Cornoil
+  name: Corn Oil
+  desc: Corn Oil, A delicious oil used in cooking. Made from corn.
```

Is this file duplicate to `Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/condiments.yml`? Otherwise:

```suggestion
  name: corn oil
  desc: Corn oil, A delicious oil used in cooking. Made from corn.
```